### ISSUE TITLE
Great difficulties with settings, postgresql connection seems ok but dead

### ISSUE THREAD

**Issue by ADWolfe-LoyNO** [mtxr/vscode-sqltools#262]
**Serious operating issues - basically can hardly get it to do anything**

I have a new install of VS Code and installed the current (2019-05-31) SQL Tools extension on Microsoft Windows 10.

In the command palette, I selected "sql tools add new connection" several times.  Usually I would get a window labeled SQL Tools Settings that was completely blank. The new connection wizard/form has only come up for me once.  Once I saw it, I was able to enter and SQL Tools said it connected correctly.

I did not see a preview of the `settings.json` file.

The database list under the SQL tools pane showed my connection, and I hit connect and the icon changed to something indicating successful connection.  However, clicking the expansion triangle did nothing.

When I entered SQL commands, I got a results window but it was completely blank - no headings, no rows, nothing.

I tried to find where the connection information was stored, so that I could manually edit it, but had no luck.

**To Reproduce**

I doubt you'll reproduce it, but...

1. install vscode 
2. install PostgreSQL using EnterpriseDB installer
3. launch vscode and install SQL Tools
4. open command palette and select add new sql tools connection
5. blank panel appears

**Expected behavior**

I expected to see the settings.json contents as on your documentation
I expected to see a list of PostgreSQL databases, schemas, tables after connecting
I expected to see results for a valid query or a useful error message
**Screenshots**

**Desktop (please complete the following information):**
 - SQLTools Version [e.g. v0.17.x]
 - VSCode Version: [e.g 1.x.y]
 - Dialect Used: [e.g PostgreSQL, Oracle, SQLite..]
 - OS: [e.g. Windows, Linux, Mac]

**Additional context**

**Comment by mtxr** [comment]
Hi, 

sorry for the late reply, I had to deal with some stuff here in the past week, I'll take a look at this within this week.

Thanks!

**Comment by mtxr** [comment]
@ADWolfe-LoyNO which SQLTools version are you using?

Would you mind sharing the logs of SQLTools output channels?

Is you DB huge?

**Comment by ADWolfe-LoyNO** [comment]
0.19.6
the database is tiny
I’m not even getting connections saved, I can’t do logs right now

I just reinstalled again, tried to create a connection, filled out the new connection and hit Create, and it went directly to another “Setup a new connection” panel.  The new information was not saved.  Repeated.  An old connection was rediscovered when I reinstalled, and it seems to connect but will not introspect the database.

- Andrew Wolfe

Andrew D Wolfe, Jr, MSCS
Assistant Visiting Professor
Monroe Hall 445
Office: +1 504 865-2951
adwolfe@loyno.edu <mailto:adwolfe@loyno.edu>


           	
Dept of Math and Computer Science
Loyola University of New Orleans

Test everything; hold fast to what is good 
— 1 Th 5:21
